src/hapi/pipelines/app/__main__.py

```diff
@@ -64,12 +64,38 @@ def parse_args():
     return parser.parse_args()
 
 
+def combine_default(country, default):
+    country["input"] = country["input"] + default["input"]
+    country["output"] = country["output"] + default["output"]
+    country["output_hxl"] = country["output_hxl"] + default["output_hxl"]
+    return country
+
+
+def inject_default(config):
+    if "default" in config:
+        default = config["default"]
+        keys = default["keys"]
```

Comment:
Can this be called something else? (Also in the config file). Like `scrapers_with_defaults` or something.